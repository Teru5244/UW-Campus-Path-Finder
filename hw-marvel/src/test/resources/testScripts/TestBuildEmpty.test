# Build up the graph from empty file and check its properties

LoadGraph empty empty.csv

ListNodes empty

FindPath empty a b