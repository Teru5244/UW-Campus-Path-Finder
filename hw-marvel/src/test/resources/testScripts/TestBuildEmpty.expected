# Build up the graph from empty file and check its properties

loaded graph empty

empty contains:

unknown: a
unknown: b